src/main/java/net/sf/jabref/logic/l10n/Languages.java

```diff
@@ -0,0 +1,32 @@
+package net.sf.jabref.logic.l10n;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Languages {
+
+    public static final Map<String, String> LANGUAGES;
+
+    static {
+        LANGUAGES = new TreeMap<>();
+
+        // LANGUAGES contains mappings for supported languages.
+        LANGUAGES.put("Dansk", "da");
+        LANGUAGES.put("Deutsch", "de");
+        LANGUAGES.put("English", "en");
+        LANGUAGES.put("Espa�ol", "es");
+        LANGUAGES.put("Persian (?????)", "fa");
```

Comment:
Thank you. I must have missed that. Somehow, Intellij has set up my project to not use UTF8 but some windows specific one.